### PR TITLE
bs - create the class for the basic MongoDB collection (admin can see subject area updates)

### PR DESCRIPTION
**Pull request by BrennaScholte**
In this pull request I created the class for the basic MongoDB collection, in Update.java. In this class I defined the three string variables: subject_area, quarter_yyyyq, last_update.

This PR is part of implementing the whole feature titled "Admins can see when a subject area was last updated".

Closes: #35

### CODE REVIEW

**Review by kkwen123 — DISMISSED** [review]
LGTM

File: src/main/java/edu/ucsb/cs156/courses/documents/Update.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Document(collection = "updates")
+public class Update {
+    private ObjectId _id;
+    private String subject_area;
+    private String quarter_yyyyq;
+    private String last_update;
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/courses/documents/Update.java:20]: This is probably better:

```suggestion
    private ZonedDateTime last_update;
```

See: Job.java under `entities` for examples (and the `import` that's needed).

**Review by pconrad — CHANGES_REQUESTED** [review]
One small change

**Review by kkwen123 — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**